src/index.html

```diff
@@ -8,7 +8,19 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
+      <h1></h1>
+      <div class="updatedTime"></div>
+      <div class="description"></div>
+      <table class="table table-striped table-bordered">
+        <tr>
+          <th>Currency</th>
+          <th>Description</th>
+          <th>Rate</th>
+          <th>Symbol</th>
+        </tr>
+      </table>
+    </div>
```

Comment:
Good work to implement the HTML structure and the final product looks very close to the design. Well done.

One thing to consider is you have a lot of empty HTML tags before the Javascript executed. Maybe we only need an empty div and create all the DOMs in JS run time? You don't have anything to show before load from the JSON file anyway.